openwisp_notifications/handlers.py

```diff
@@ -373,3 +379,31 @@ def invalidate_cache():
 
     # execute cache invalidation only after changes have been committed to the DB
     transaction.on_commit(invalidate_cache)
+
+
+@receiver(user_logged_in)
+def check_email_verification(sender, user, request, **kwargs):
+    admin_path = reverse('admin:index')
+    # abort if this is not an admin login
+    if not user.is_staff or not request.path.startswith(admin_path):
```

Comment:
@SitaGanesh notice that I also changed the code slightly to avoid too many nested if statements which make the code less readable. I also added a few comments which should help maintainers read the code faster.